src/components/Link/__stories__/examples/LinkExampleView/LinkExampleView.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import {Link} from '../../..';
+import {Text} from '../../../../Text';
+import {DocsExample} from '../../../../../demo/DocsExample/DocsExample';
+
+export function LinkExampleViewNormal() {
+    return (
+        <DocsExample gap="m">
```

Comment:
`gap="m"` is default

Reply:
I took this from the button stories, deleted it here